### PR TITLE
Add script to download CAMS

### PR DESCRIPTION
**Pull request by lewisjared**
## Description
Downloads CH4 data on pressure levels from CAMS. These data are used by BCON to define the boundary conditions for CMAQ.

## Checklist

Please confirm that this pull request has done the following:

- [x] Tests added
- [ ] Documentation added (where applicable)
 

## Notes

@prayner Does this need to be performed on every run or is this just for bootstrapping? I'm asking because the CAMS reanalysis data is only available up til 2024-01-01.

### CODE REVIEW

**Review by mikapfl — APPROVED** [review]

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -51,6 +51,21 @@ jobs:
...
+        with:
+          path: data/inputs
+          key: cams
+      - name: Download CAMS input data
+        if: steps.cache-cams.outputs.cache-hit != 'true'
+        run: |
+          # This can take a few minutes as the CAMS data is stored on tape
+          docker run --rm \
+            -v ${{ github.workspace }}:/project \
+            -e CDSAPI_KEY=${{ secrets.CDSAPI_KEY }} \
```
[the remark this answers is not in the document]
Comment (crdanielbusch) [reply .github/workflows/build.yaml:64]: Good to know! Are any change in the code required for that? Or will it automatically detect the api key in the environment variable?

[the remark this answers is not in the document]
Comment (lewisjared) [reply .github/workflows/build.yaml:64]: The thing to know is that the environment variable takes precedence over the config file. If you don't hard code the URL like here, you would also need the `CDSAPI_URL` environment variable.

No other changes needed

**Review by crdanielbusch — APPROVED** [review]
